integration/maptest/main_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package maptest
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/google/trillian/testonly/integration"
+)
+
+var (
+	server = flag.String("map_rpc_server", "", "Server address:port")
+	env    *integration.MapEnv
```

Comment:
`TestMain` exists to perform test setup / teardown, but it can't pass variables to the individual test functions except through package variables.

I think `env` is ok as a package variable because it is not specific to each test - it just contains connection information to the Trillian Map Server. Tests are separated by using individual trees.